pyfolio/utils.py

```diff
@@ -157,11 +157,23 @@ def get_returns_cached(filepath, update_func, latest_dt, **kwargs):
         returns = update_func(**kwargs)
         try:
             mkdir(cache_dir())
+        except OSError as e:
```

Comment:
This probably wants to be a function.  We use this in the Q codebase downstream:

```
import os
from errno import EEXIST


def ensure_directory(path):
    """
    Ensure that a directory named "path" exists.
    """
    try:
        os.makedirs(path)
    except OSError as exc:
        if exc.errno == EEXIST and os.path.isdir(path):
            pass
        else:
            raise
```